core/src/main/resources/db/changelog/changesets/2023_05_15_kit_request.yaml

```diff
@@ -0,0 +1,17 @@
+databaseChangeLog:
+  - changeSet:
+      id: kit_request
+      author: breilly
+      changes:
+        - createTable:
+            tableName: kit_request
+            columns:
```

Comment:
BLOCKER: we need a "creating_admin_user_id" field here for auditability.  For now, it should be non-nullable.  Later, when we add rule-based auto-kit sending, we can modify that constraint to say "must have either an admin id, or a rule_source specified"